theano/gpuarray/ROIPoolGPUFwd.c

```diff
@@ -1,9 +1,9 @@
 #section kernels
 
-#kernel ROIPoolGPUFwd_kernel : ssize, ssize, *, ssize, ssize, ssize, ssize, ssize, ssize, *, *, * :
+#kernel ROIPoolGPUFwd_kernel : int32, int32, *, float32, int32, int32, int32, int32, int32, *, *, * :
```

Comment:
Thank you so much for this change. Are these options for variable type documented somewhere? I would like to add it otherwise :)